news/2020-07-06-renewed-beatmap-spotlights-summer-2020.md

```diff
@@ -0,0 +1,68 @@
+- There will be **four playlists in total, consisting of 5 beatmaps** and each being played three times during the season. This gives everyone plenty of opportunities to revisit their weekly scores as only your best total score on a playlist will be considered for your position on the Seasonal Ranking.
+- On top of that **the Top 10 people of each timeshift lobby will be awarded with 1 week of supporter tag**.
+- At the end of the season, we will **elect the best mapper of this season** and provide a **new unlockable medal** for those who want to play through the Seasonal Beatmaps once the season has concluded.
+
+Of course each game mode will have their own leaderboard and playlists. If you are interested in the full details of this system, you can visit the [Beatmap Spotlights wiki page](/wiki/Beatmap_Spotlights) to learn more.
+
+## Season Schedule
+
+With that being said, the Summer 2020 season has been fully prepared and is ready for deployment. Therefore, the first timeshift lobby will open **this Friday** already.
+
+| Event | Time |
+| --: | :-- |
+| Season Start | Jul 7th 2020 |
+| Timeshift Lobby: Playlist A | Jul 10th - Jul 12th 2020 |
+| Timeshift Lobby: Playlist B | Jul 17th - Jul 19th 2020 |
+| Initial Profile Badge Distribution | Jul 21st - Jul 23rd 2020 |
+| Timeshift Lobby: Playlist C | Jul 24th - Jul 26th 2020 |
+| Timeshift Lobby: Playlist D | Jul 31st - Aug 2nd 2020 |
+| Timeshift Lobby: Playlist A | Aug 7th - Aug 9th 2020 |
+| Timeshift Lobby: Playlist B | Aug 14th - Aug 16th 2020 |
+| Timeshift Lobby: Playlist C | Aug 21st - Aug 23rd 2020 |
+| Timeshift Lobby: Playlist D | Aug 28th - Aug 30th 2020 |
+| Timeshift Lobby: Playlist A,B | Sep 4th - Sep 6th 2020 |
+| Timeshift Lobby: Playlist C,D | Sep 11th - Sep 13th 2020 |
+| Season Conclusion | Sep 14th - Sep 18th 2020 |
+
+## Rewards
+
+Depending on your position on the Seasonal Ranking, you will be awarded with a unique profile badge. Your exact position will only be known to you when you are in the Top 50.
+
+| Badge | Bracket Tier | Placement |
+| :--: | --: | :-- |
+| ![](/wiki/Beatmap_Spotlights/img/Rhythm_Incarnate.jpg) | Rhythm Incarnates | Top 50 |
+| ![](/wiki/Beatmap_Spotlights/img/Diamond.jpg) | Diamond | Top 51 - 3% |
+| ![](/wiki/Beatmap_Spotlights/img/Platinum.jpg) | Platinum | 3% - 10% |
+| ![](/wiki/Beatmap_Spotlights/img/Gold.jpg) | Gold | 10% - 25% |
+| ![](/wiki/Beatmap_Spotlights/img/Silver.jpg) | Silver | 25% - 50% |
+| ![](/wiki/Beatmap_Spotlights/img/Bronze.jpg) | Bronze | 50% - 70% |
+| ![](/wiki/Beatmap_Spotlights/img/Copper.jpg) | Copper | 70% - 95% |
+| ![](/wiki/Beatmap_Spotlights/img/Iron.jpg) | Iron |  95% - 100% |
+
+## Winter Spotlights results
+
+We did not forgot the winners of the Winter Beatmap Spotlights 2020. Congratulations to the winners! They will receive 3, 2, and 1 month of osu!supporter for 1st, 2nd, and 3rd place respectively for their efforts!
```

Comment:
Maybe swap second use of winners with "victors" or some other synonym to avoid repetition?